packages/common/src/constants/FeatureFlags.ts

```diff
@@ -56,7 +56,7 @@ export const FeatureFlags = {
       SceneComplexityNotification: 'ir.editor.ui.sceneComplexityNotification',
       TransformPivot: 'ir.editor.ui.transformPivot',
       Hierarchy: {
-        ShowModelChildren: 'ir.editor.ui.hierarchy.showModelChildren'
+        HideGlbChildren: 'ir.editor.ui.hierarchy.showModelChildren'
```

Comment:
we spoke on discord and agreed revert the renaming such that we don't have to put a PR in to multitenancy